--- itk/Makefile.in.orig	2003-12-31 17:35:30.000000000 +0100
+++ itk/Makefile.in	2003-12-31 18:28:13.290461032 +0100
@@ -80,7 +80,7 @@
 # library that we are building.
 #========================================================================
 
-PKG_LIB_FILE	= @PKG_LIB_FILE@
+PKG_LIB_FILE	= @PKG_LIB_FILE@.0.0
 PKG_STUB_LIB_FILE = @PKG_STUB_LIB_FILE@
 
 lib_BINARIES	= $(PKG_LIB_FILE)
@@ -315,8 +315,8 @@
 
 $(PKG_LIB_FILE): $(PKG_OBJECTS)
 	-rm -f $(PKG_LIB_FILE)
-	${MAKE_LIB}
-	$(RANLIB) $(PKG_LIB_FILE)
+	${MAKE_LIB} -Wl,-soname,libitk$(PACKAGE_VERSION).so.0
+	-ln -sf $(PKG_LIB_FILE) libitk$(PACKAGE_VERSION).so.0
 
 $(PKG_STUB_LIB_FILE): $(PKG_STUB_OBJECTS)
 	-rm -f $(PKG_STUB_LIB_FILE)
--- itk4.1.0/pkgIndex.tcl.in.orig	2019-01-17 19:22:30.327974793 +0100
+++ itk4.1.0/pkgIndex.tcl.in	2019-01-17 19:57:16.177950975 +0100
@@ -2,5 +2,5 @@
 
 if {![package vsatisfies [package provide Tcl] 8.6]} return
 if {[string length [package provide Itcl]] && ![package vsatisfies [package provide Itcl] 4.1]} return
-package ifneeded itk @PACKAGE_VERSION@ [list load [file join $dir "@PKG_LIB_FILE@"] Itk]
-package ifneeded Itk @PACKAGE_VERSION@ [list load [file join $dir "@PKG_LIB_FILE@"] Itk]
+package ifneeded itk @PACKAGE_VERSION@ [list load "@libdir@/@PKG_LIB_FILE@.0" Itk]
+package ifneeded Itk @PACKAGE_VERSION@ [list load "@libdir@/@PKG_LIB_FILE@.0" Itk]
