--- itk/configure.in.orig	2011-11-28 19:12:31.000000000 +0100
+++ itk/configure.in	2011-12-01 18:05:05.737054766 +0100
@@ -65,6 +65,9 @@
 # the basic setup necessary to compile executables.
 #------------------------------------------------------------------------
 
+dnl the above is no longer true wrt. AC_PROG_INSTALL
+AC_PROG_INSTALL
+
 TEA_SETUP_COMPILER
 
 #-----------------------------------------------------------------------
